<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>252</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="boton_imagmed">
      <property name="text">
       <string>Trabajar con Imágenes Médicas (DICOM/NIfTI)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_imagconven">
      <property name="text">
       <string>Trabajar con Imágenes Convencionales (JPG/PNG)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boton_salir_imagenes">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
